src/lambdaserver.ts

```diff
@@ -246,7 +302,14 @@ export default class LambdaServer extends Webda {
         ctx.statusCode = err;
         this.flushHeaders(ctx);
       } else {
-        this.log("ERROR", err);
+        this.log(
```

Comment:
The only change is there......

Reply:
We should not display body in the log as it might contain sensitive data, the vhost is a concept that has been dropped, webda was able to serve several hosts with same application but serverless/docker makes it more logical to use 2 deployments